CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- **BREAKING:** Bump dependency on `@metamask/network-controller` to ^13.0.0 ([#191](https://github.com/MetaMask/smart-transactions-controller/pull/191))
```

Comment:
not that it matters since this will be a major dep bump anyways but why is this breaking in the context of this package?

Reply:
Great catch. I guess technically it isn't breaking because it doesn't directly use the breaking changes from NetworkController. Only thing referenced is the `NetworkState` type, and it doesn't touch `networkId` there. I'll remove the BREAKING

Reply:
Done: https://github.com/MetaMask/smart-transactions-controller/pull/191/commits/85b5b7efbdf8c1431f72ff8f8554f71e7c990a2f